common/setup.py

```diff
@@ -25,18 +25,26 @@ def get_changelog():
     setup(
       name="devpi-common",
       description="utilities jointly used by devpi-server and devpi-client",
-      long_description="\n\n".join([README, CHANGELOG]),
+      long_description="\n\n".join([README]),
       version='3.4.0',
       packages=find_packages(),
       install_requires=["requests>=2.3.0", "py>=1.4.20"],
       url="https://github.com/devpi/devpi",
       maintainer="Holger Krekel",
       maintainer_email="holger@merlinux.eu",
+      license='MIT',
       classifiers=[
         "Development Status :: 5 - Production/Stable",
         "Environment :: Web Environment",
         "Intended Audience :: Developers",
         "Intended Audience :: System Administrators",
         "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2.6",
```

Comment:
The Python classifiers you added don't reflect the current state. There are long obsolete versions and others like 3.6 are missing.

Reply:
> I don't see how these changes have anything to do with building rpms. If you could explain how these help, we can find a way to fix this properly.

Yes, It's my bad. Building package or creating rpm has nothing to do with it. However, I think improvement is needed in the rpm spec.

Reply:
> The Python classifiers you added don't reflect the current state. There are long obsolete versions and others like 3.6 are missing.

Agree, I didn't go through the entire log to see all supported version, Yes will include 3.6 for sure